src/promptflow/promptflow/_core/tools_manager.py

```diff
@@ -90,6 +91,7 @@ def collect_package_tools(keys: Optional[List[str]] = None) -> dict:
                 importlib.import_module(m)  # Import the module to make sure it is valid
                 tool["package"] = entry_point.dist.metadata["Name"]
                 tool["package_version"] = entry_point.dist.version
+                assign_tool_input_index_for_ux_order_if_needed(tool)
```

Comment:
please make sure you have tests that can verify this change.

Reply:
Added test, thanks.